dali/python/nvidia/dali/ops.py

```diff
@@ -581,6 +581,53 @@ def _repack_list(self, sets, fn):
     # Operator.__call__.__doc__ = _docstring_generator_call(name)
     return Operator
 
+
+def _to_snake_case(camel):
+  out = ""
+  nupper = 0
+  start = 0
+  for i, c in enumerate(camel):
+    if c.isupper():
+      if nupper == 0:
+        start = i
+      nupper += 1
+    else:
+      if nupper == 0:
+        out += c
+      else:
+        if len(out) > 0:
+          out += '_'
+        if nupper > 1:
+          out += camel[start:i-1].lower() + '_'
+        out += camel[i-1].lower()
+        out += c
+        nupper = 0
+
+  if nupper > 0:
+    out += camel[start:i].lower()
+  out = out.replace("b_box", "bbox").replace("mx_net", "mxnet")
```

Comment:
I think we should have a dict of special cases in one place. Here is easy to miss and extend.

Reply:
Yes and no. BBox is not a special case - rather a pattern. With MXNet - currently we have just one operator, but should we have more, it's already handled.